lucene/CHANGES.txt

```diff
@@ -54,7 +54,7 @@ Bug Fixes
 
 Other
 ---------------------
-(No changes)
+* LUCENE-9976: Fix WANDScorer assertion error. (Zach Chen, Adrien Grand, Dawid Weiss)
```

Comment:
Just curious...it seems there are 2 `Lucene 8.10.0` sections above. Is that done for some reason?

Reply:
It's a mistake indeed, I just removed the extra one!

Reply:
Sounds good, thanks Adrien!